libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2634,6 +2634,17 @@ MAV_RESULT GCS_MAVLINK::handle_preflight_reboot(const mavlink_command_long_t &pa
 
     // send ack before we reboot
     mavlink_msg_command_ack_send(chan, packet.command, MAV_RESULT_ACCEPTED);
+
+    //Forward this message to ACE by finding the channel that ACE is connected to
+    for(int i=0; i<GCS::get_singleton()->num_gcs(); ++i) {
+        GCS_MAVLINK* chan_ = GCS::get_singleton()->chan(i);
+        if(chan_->get_interval_for_stream(STREAM_PLANCK) > 0) {
```

Comment:
Just curious about this `get_interval_for_stream` , what is it exactly ?

Reply:
Its the streamrate of the PLANCK_STATEINFO messages as requested by ACE.  Basically, if this is zero, theres probably not an ACE system connected on this channel.  If its non-zero, there definitely _is_ an ACE system connected on this channel.

Reply:
Got it thanks 👍